Tools/machines/frontier-olcf/submit.sh

```diff
@@ -31,6 +31,11 @@
 # or, less invasive:
 export FI_MR_CACHE_MONITOR=memhooks  # alternative cache monitor
 
+# Seen since August 2023
+# OLCFDEV-1597: OFI Poll Failed UNDELIVERABLE Errors
+# https://docs.olcf.ornl.gov/systems/frontier_user_guide.html#olcfdev-1597-ofi-poll-failed-undeliverable-errors
+export MPICH_SMP_SINGLE_COPY_MODE=NONE
```

Comment:
According to our earlier ticket in Summer, OLCFHELP-12850, we were recommended at the time to also set

```bash
export FI_CXI_RX_MATCH_MODE=software
```

We saw no improvement setting the three env vars (this PR + the one above) at the time. `FI_CXI_RX_MATCH_MODE` is not mentioned in the OLCF docs at this time.

Reply:
My last test for this was on June 7th which had all 3 options and it ran for 12 hours without failing.
That was on 384 nodes and `export FI_CXI_RX_MATCH_MODE=software` was the one that was newly added. However, I did not have the time to test if this was reproducible. Apologies! :pray:

Reply:
Otherwise, the issue appeared randomly throughout the simulations and it would run fine for hours sometimes.

Reply:
Will add!